docker-entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e
+
+if [ "$1" = '/ucentral/ucentralgw' -a "$(id -u)" = '0' ]; then
+    chown -R "$UCENTRALGW_USER". "$UCENTRALGW_ROOT" "$UCENTRALGW_CONFIG"
```

Comment:
What for is the dot after UCENTRALGW_USER?

Reply:
This is to apply the same group ownership (basically just leaving the group blank). But it seems that the dot is an old way of separating the user and group fields and chown just supports it out of legacy reasons, so I think I'll change it to a colon.